src/fs.rs

```diff
@@ -250,7 +250,7 @@ fn symlink_attr(ino: u64) -> FileAttr {
         crtime: UNIX_EPOCH,
         uid: 0,
         gid: 0,
-        perm: 777,
+        perm: 0o555,
```

Comment:
I have to see if this is something that I can do by default or if this should be rather hidden behind a flag.